From 3b3f12df5d2753fba5bd63e5f70b02324a3fcc69 Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Sun, 31 Oct 2021 18:45:57 +0200
Subject: [PATCH] [SW-0] onix,vivado: moving to fpga v35

Need to upgrade the FPGA version for the regression test.
Adds a tcl file to load the v35 bitstream file.
Point onix-programming.sh script to invoke the v35 tcl script

Change-Id: I3317abf5eeae33ff24bb23d12c9352f10a5bbe7d
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/scripts/vivado_lab/onix-programming/onix-program.sh b/scripts/vivado_lab/onix-programming/onix-program.sh
index 8ad125d..01ce7b1 100755
--- a/scripts/vivado_lab/onix-programming/onix-program.sh
+++ b/scripts/vivado_lab/onix-programming/onix-program.sh
@@ -3,7 +3,7 @@
 #viv -mode batch -source script.tcl -tcargs
 HL_PASSWD=Hab12345
 HOST_NAME=emb-srv-host17
-hl_version=34
+hl_version=35
 wait_in_secs=50
 wait_to_host_in_secs=90
 host_num=120
diff --git a/scripts/vivado_lab/onix-programming/program-avnet-35.tcl b/scripts/vivado_lab/onix-programming/program-avnet-35.tcl
new file mode 100644
index 0000000..2382a17
--- /dev/null
+++ b/scripts/vivado_lab/onix-programming/program-avnet-35.tcl
@@ -0,0 +1,9 @@
+open_hw
+connect_hw_server
+open_hw_target
+current_hw_device [get_hw_devices xcvu440_0]
+refresh_hw_device [lindex [get_hw_devices xcvu440_0] 0]
+set_property PROGRAM.FILE {./h6_top_v35_141021.bit} [get_hw_devices xcvu440_0]
+program_hw_devices [get_hw_devices xcvu440_0]
+refresh_hw_device [lindex [get_hw_devices xcvu440_0] 0]
+exit
